packages/build/tests/error/snapshots/tests.js.md

```diff
@@ -1705,7 +1705,7 @@ Generated by [AVA](https://ava.li).
     Error monitoring payload:␊
     {␊
       "errorClass": "dependencies",␊
-      "errorMessage": "A Netlify Function failed to require one of its dependencies./nPlease make sure it is present in the site's top-level /"package.json/"./n/nIn file /"packages/build/tests/error/fixtures/dependencies/functions/test.js"/external/path find module 'does-not-exist'/external/path stack:/n- /node_module/path /node_module/path /node_module/path packages/build/src/plugins_core/functions/index.js/n- packages/build/src/commands/get.js/n- packages/build/src/core/main.js/n- packages/build/tests/helpers/main.js/n- packages/build/tests/error/tests.js/n- /node_module/path",␊
+      "errorMessage": "A Netlify Function failed to require one of its dependencies./nPlease make sure it is present in the site's top-level /"package.json/"./n/nIn file /"packages/build/tests/error/fixtures/dependencies/functions/test.js"/external/path find module 'does-not-exist'/external/path stack:/n- /node_module/path /node_module/path /node_module/path packages/build/src/plugins_core/functions/utils.js/n- packages/build/src/plugins_core/functions/index.js/n- packages/build/src/commands/get.js/n- packages/build/src/core/main.js/n- packages/build/tests/helpers/main.js/n- packages/build/tests/error/tests.js/n- /node_module/path",␊
```

Comment:
The stack trace changed because we've introduced a new file (`functions/utils.js`).